site/content/en/docs/Reference/gameserverallocation.md

```diff
@@ -7,6 +7,10 @@ description: "A `GameServerAllocation` is used to atomically allocate a GameServ
 weight: 30
 ---
 
+{{% feature publishVersion="1.38.0" %}}
```

Comment:
No need for a feature flag - feature flags are only for when we talk about a feature that hasn't been released yet. Updated/improvements to existing docs for existing features don't need it.